addon/components/o-s-s/information-section.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@glimmer/component';
+import type { IconNames } from './iconName.enum';
+
+interface InformationSectionArgs {
+  badgeIcon?: keyof typeof IconNames;
+  svgIcon?: string;
```

Comment:
`imageUrl` should be a bit more "permissive" language wise :) 
the icons might be pngs sometimes

Reply:
Changed to imageUrl